docs/concepts/client/send-receive-data.md

```diff
@@ -196,6 +196,17 @@ with f:  # Using it as a Context Manager will start the Flow
 This will send the request to all Executors whose names start with 'bar', such as 'barExecutor'.
 In the simplest case, you can specify a precise Executor name, and the request will be sent only to that single Executor.
 
+## Use Unary or Streaming gRPC
+
+The Flow with **gRPC** protocol implements the unary and the streaming RPC lifecycle for communicating with the clients. When sending more than one request using the batching or the iterator mechanism, the RPC lifecycle for the {meth}`~jina.clients.mixin.PostMixin.post` method can be controlled using the `stream` boolean method argument. By default the stream option is set to `False` which uses the unary RPC to send the data to the Flow. If the stream option is set to `True` the streaming RPC is used to send the data to the Flow.
```

Comment:
I would mention that behavior should not change

Reply:
I provided the link to the grpc documentation in the `hint` section. Should I be more explicit?

Reply:
good then